### ISSUE TITLE
Update admin privileges

### ISSUE THREAD

**Issue by jacktilford** [jacktilford/dctc-pickle#5]
(from Jim Sterken) A new request: would you please give MikeB, JohnL, MikeW, DebbieW, and me a higher level of rights so we can each delete court bookings belonging to other people?  This came up a few days ago when I was talking to MikeB.  Someone came up and wanted us to know they had reserved a court but could not figure out how to delete the reservation.  I expect that comes up regularly and it would be nice during crowded times if some extra people could correct bad bookings.

**Comment by jacktilford** [comment]
Administrators (MikeW, DebbieW, JackT) already have the right to delete reservations in both the website and the app. In the app, from the admin panel, go to "reservations -> courts" and then navigate to the reservation in question. There is a big red cancel button that does the job. In the website, it's "reservations -> list", select the "courts" tab, find the reservation, and click on the black X at the end of the line.

**Comment by jacktilford** [comment]
For MikeB, JohnL, and JimS: it is possible to create a "sub-admin" user and grant a variety of privileges, including the ability to cancel a reservation. Unfortunately there is no way to mark an existing user as a sub-admin. Instead, one must create a new sub-admin user with the same email as an existing users, and the system links the two. I did this for Jim and it seems to work; his name appears only once in the member list. I will do the same for Mike and John after Jim confirms he can delete reservations.

**Comment by jimsterken** [comment]
Hello Jack,
![NotAllowedToEdit](https://user-images.githubusercontent.com/31393805/220408367-b3f849cf-e656-4f25-a149-7c37b2dfb463.jpg)

I asked Gwynn to make a reservation today at 7pm on Court 5.  She did that, but I am not allowed to edit it.  See the screenshot.  I tried both from Windows./Chrome and from my iPhone.

**Comment by jimsterken** [comment]
Starting today, I'm getting notifications when new players ore events are added.  I suspect this is related to giving me extra admin rights.  However, I just checked, and I cannot delete Gwynn's 7pm reservation for tonight.  I've attached the notification message I'm receiving now.
![NewNotificationEmail](https://user-images.githubusercontent.com/31393805/220498973-d4935806-6715-46d1-81cc-f40a9ac12fdb.jpg)

**Comment by jacktilford** [comment]
OK, I turned off notifications for you (I presume that's what you want - you will get inundated otherwise). My morning is full, but this afternoon I'll figure out why you can't delete reservations.

Jack

**Comment by jimsterken** [comment]
Jack, thanks.

No rush on making these changes.  I'm just trying to report the glitches I
see in concise ways so you can address them whenyou have time.

   Jim

**Comment by jimsterken** [comment]
Jim,
Did you try to edit the reservation or to delete it? I had only added delete privileges; I’ve now also added edit privileges.
Jack

==============

Jack, I tried CourtReserve on both my PC in Chrome and in the CR app on my iPhone.  In each case, I clicked Reservations then moved to where I could see Gwynn's 7 pm reservation.  Then I clicked or touched the reservation expecting a pop-up to emerge with a choice to edit or delete.  Nothing happened.  I get the same behavior today -- I just checked to see if I could modify or delete the singles reservation at 3pm Friday by Rebecca Blankfield.  Nothing happened when I clicked, double-clicked, etc.   

Am I supposed to do that in a different way?

**Comment by jacktilford** [comment]
Please try this (this is the writeup I put in the github issue):

"In the app, from the admin panel, go to "reservations -> courts" and then navigate to the reservation in question. There is a big red cancel button that does the job. In the website, it's "reservations -> list", select the "courts" tab, find the reservation, and click on the black X at the end of the line."

Jack

**Comment by jimsterken** [comment]
Please try this (this is the writeup I put in the github issue):

"In the app, from the admin panel, go to "reservations -> courts" and then navigate to the reservation in question. There is a big red cancel button that does the job. In the website, it's "reservations -> list", select the "courts" tab, find the reservation, and click on the black X at the end of the line."

Jack

======

Jack, where is the admin panel?  I don't see any way to get to an admin panel on the iPhone app.  Also I can't find an admin section on the PC browser version of CR.  My login in jsterken@gmail.com

**Comment by jacktilford** [comment]
On the website, do you see this when you hover over your name in the upper right (here I'm using a test account with sub-admin privileges, set up just like yours):

***@***.***

If so, click on "Admin Panel" and you should see this:

***@***.***

Now click on List and you will see a list of reservations (with an "X" at the end of the line for cancellation).

As for the phone app, CourtReserve is giving me no end of trouble, and will not let me log in there with my test account. Let's take a look at your phone this afternoon at 1:00.

**Comment by jimsterken** [comment]
Yes, I do.  That's probably it.  I have to go offline for an hour or so now.

**Comment by jimsterken** [comment]
Hi Jack,
  I'm back online now.  The Admin Panel works from my browser.

  What about the iPhone?  I don't see a way to switch into admin mode there.

     Jim
